app-misc/fdutils/fdutils-5.6_p2.ebuild

```diff
@@ -39,11 +39,13 @@ src_prepare() {
 		"${FILESDIR}"/fdutils-5.5.20060227-r1-parallel.patch # bug 315577
 		"${FILESDIR}"/fdutils-5.6_p2-parallel.patch
 		"${FILESDIR}"/fdutils-5.6_p2-docs-build.patch
+		"${FILESDIR}"/fdutils-5.6_p2-variable-ar.patch
 	)
 
 	default
 
 	eautoreconf
+	touch ar-lib || die # bug 834874
```

Comment:
I learned this trick from

 - https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=068c5c161ddc063253e8f827c40a4ceace5fa3f5
 - https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=1124a7c8003ab7a89448842698c834a3a6154600

hope it's still the correct thing to do :-)

Reply:
It's cheesy but it'll do ;)

Reply:
Great!